### ISSUE TITLE
Widget is not accessible through keyboard navigation

### ISSUE THREAD

**Issue by ariesbrunetyson** [720kb/angular-datepicker#7]
I tried to access the element using keyboard alone. Seems like i couldn't able to access/select the date from the picker. And when the focus moves to the next element the datepicker from the previous is still on.
![image](https://cloud.githubusercontent.com/assets/6047770/4953900/b555d4ac-6686-11e4-867a-4a3b79868c8d.png)

**Comment by wouldgo** [comment]
more than 1 month w/o any feedback. Closing.
